conf/templates/peach_go_sbot/DEBIAN/postinst

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+set -e
+
+# create user which go-sbot runs as
+adduser --quiet --system peach-go-sbot
```

Comment:
This is so nice <3 Being able to create the user and set binary permissions right here is awesome. Great find! This is going to make our whole installation and system maintenance process so much simpler.